system/filesystem.py

```diff
@@ -63,17 +66,68 @@ def main():
             dev=dict(required=True, aliases=['device']),
             opts=dict(),
             force=dict(type='bool', default='no'),
+            resizefs=dict(type='bool', default='no'),
         ),
         supports_check_mode=True,
     )
 
+    # There is no "single command" to manipulate filesystems, so we map them all out and their options
```

Comment:
You can use fsadm for fs-check and resize.
It supports ext2, ext3, ext4, ReiserFS and XFS and in near feature btrfs.

fsadm -y resize /dev/vg/test